build/dependencies.yaml

```diff
@@ -1,4 +1,20 @@
 dependencies:
+  # agnhost: bump this one first
+  - name: "agnhost"
+    version: "2.22"
+    refPaths:
+    - path: test/images/agnhost/VERSION
+      match: \d.\d
+    - path: test/images/agnhost/agnhost.go
+      match: "Version:"
+
+  # then after merge and successful postsubmit image push / promotion, bump this
+  - name: "agnhost: dependents"
```

Comment:
I suspect there are more, but I didn't turn up any that are obviously intended to be in sync here yet.
If nothing else though, this entry should prevent an incorrect addition of this file to the entry above.

There is also the rarely updated private image from this that we literally only use to test pulling a private image (w/ auth).